particles/Processing/Dispose.particle

```diff
@@ -0,0 +1,5 @@
+import './schemas/MlResource.schema'
+
+particle Dispose in './js/Dispose.js'
+  in * {Number ref} resource
+  //description `Clean up resources.`
```

Comment:
Why don't we want a description here?

Reply:
These descriptions are for building into Suggestions for the end-user.

Reply:
Also, I forgot to put this in my notes ... the DescriptionFormatter will attempt to form an English sentence out of descriptions, so you want to avoid (1) capitals (unless for a proper-name) and (2) ending punctuation (e.g., a corrected description would be `clean up resources`.)

Reply:
Good notes, thank you.